packages/fxa-content-server/app/scripts/views/mixins/cwts-on-signup-password.js

```diff
@@ -101,29 +93,31 @@ export default {
   },
 
   beforeSubmit() {
-    if (this.isCWTSOnSignupPasswordEnabled()) {
-      const offeredSyncEngines = this._getOfferedEngineIds();
-      const declinedSyncEngines = this._getDeclinedEngineIds();
-      const enabledSyncEngines = offeredSyncEngines.filter(
-        e => declinedSyncEngines.indexOf(e) === -1
-      );
+    if (!this.isCWTSOnSignupPasswordEnabled()) {
+      return;
+    }
 
-      // Tell multi-service browser integrations whether to send
-      // the offered and declined sync engines.
-      if (declinedSyncEngines.length === offeredSyncEngines.length) {
-        this.disableSync();
-      } else {
-        this.enableSync();
-      }
+    const offeredSyncEngines = this._getOfferedEngineIds();
+    const declinedSyncEngines = this._getDeclinedEngineIds();
+    const enabledSyncEngines = offeredSyncEngines.filter(
+      e => declinedSyncEngines.indexOf(e) === -1
+    );
+
+    // Tell multi-service browser integrations whether to send
+    // the offered and declined sync engines.
+    if (declinedSyncEngines.length === offeredSyncEngines.length) {
```

Comment:
Pretty verbose put perhaps a more clear way of indicating "all offered sync engines were declined": `offeredSyncEngines.every(e => declinedSyncEngines.includes(e))` - by no means a blocker though.

Reply:
The other way was a bit clearer to me heh, I'll keep for now.